pyuvsim/source.py

```diff
@@ -157,15 +156,19 @@ def coherency_calc(self, telescope_location):
 
             polarized_sources = np.where(~Ionly_mask)[0]
             sinX = np.sin(self.hour_angle)
-            cosX = np.tan(telescope_location.lat) * np.cos(self.dec) - np.sin(self.dec) * np.cos(self.hour_angle)
+            cosX = np.tan(telescope_location.lat) * np.cos(self.dec) - np.sin(self.dec) * np.cos(
+                self.hour_angle)
```

Comment:
I'd prefer a break just before the minus sign. Easier to read the math that way.

Reply:
agreed. fixed.